Dockerfile

```diff
@@ -52,9 +55,17 @@ RUN apt-get update && \
         && \
     rm -rf /var/lib/apt/lists/*
 
+COPY --from=builder \
+     mbsolr-jaxb/target/mbsolr-jaxb-1.0-SNAPSHOT-jar-with-dependencies.jar \
+     /opt/solr/server/lib/
+
 COPY --from=builder \
      mb-solr/target/mb-solr-0.0.1-SNAPSHOT-jar-with-dependencies.jar \
-     /opt/solr/lib/
+     /opt/solr/server/solr/lib/
```

Comment:
The README file in solr says that to add a jar to solr's class path, you can either set `solr.sharedLib` _or_ add it directly to `server/solr/lib` where it will be picked up. The documentation seemed to indicate that this was the "preferred" behaviour, so I changed it. If we want to keep this, we should delete the `ENV SOLR_OPTS="$SOLR_OPTS -Dsolr.sharedLib=/opt/solr/lib"` line later in this file.